File: DSA Love Babbar/08_Basic_Maths/P1_count_primes.py
                      
            
def count_primes(n):
    count = 0
    prime = [True] * (n + 1)

    prime[0] = prime[1] = False

    for i in range(2, n):
        if prime[i]:
            count += 1
            for j in range(2 * i, n, i):
                prime[j] = False

    return count

File: DSA Love Babbar/D008_Basic_Maths/P1_count_primes.js
function countPrimes(n) {
    let count = 0;
    let prime = new Array(n + 1).fill(true);
  
    prime[0] =[1] = false;
  
    for (let i = 2; i < n; i++) {
      if (prime[i]) {
        count++;
        for (let j = 2 * i; j < n; j += i) {
          prime[j] = false;
        }
      }
    }
  
    return count;
  }